DSTV.Net/Implementations/BodyReader.cs

```diff
@@ -17,6 +17,7 @@ public static async Task<IEnumerable<DstvElement>> GetElementsAsync(ReaderContex
             foreach (var holeNote in holeBlocks.SelectMany(holeList => holeList))
                 try
                 {
+                    if (holeNote.Equals(ContourType.BO.ToString(), StringComparison.Ordinal)) continue;
```

Comment:
I'd prefer the use of the nameof operator in these cases (e.g. `nameof(ContourType.BO)`), but that's pretty personal...

Reply:
I agree

Reply:
Me too!